src/components/layout/footer/footer.tsx

```diff
@@ -1,43 +1,33 @@
 import { Box, Container, Flex, Grid, Link, Text } from 'theme-ui';
-import { Booking } from '../../booking';
-import { Logo } from '../../logo';
-import { NetworksLinks } from './networks-links';
 
 export const Footer: React.FC = () => (
-  <Box sx={{ minHeight: '5vh', background: 'black80', mt: 'auto' }}>
-    <Container variant="readable">
+  <Box sx={{ minHeight: '5vh', background: '#0F2040', mt: 'auto' }}>
```

Comment:
Why are we setting a hex color here? We should use a value from the theme

Reply:
Fixed!